justice/datasets/plasticc_data.py

```diff
@@ -151,6 +203,18 @@ def _sqlite_get_lc_by_target(cls, conn, target):
         obj_ids = conn.execute(q, [target]).fetchall()
         return [cls._sqlite_get_lc(conn, 'training_set', o) for (o, ) in obj_ids]
 
+    @classmethod
+    def _bcolz_get_lc_by_target(cls, source, target):
```

Comment:
I think read_table is not free, so I guess I'd advocate for an API that encourages reuse of such objects (maybe just wrap them into a namedtuple or something)